libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -1,9 +1,14 @@
+#ifndef MQTT_LIGHTWEIGHT_H
+#define MQTT_LIGHTWEIGHT_H
+
 #include <stdbool.h>
 #include <stddef.h>
 #include <stdint.h>
 
 #include "config.h"
 
+#define MQTT_PACKET_TYPE_CONNECT         ( ( uint8_t ) 0x10U )
+
 struct MQTTFixedBuffer;
```

Comment:
Where are these defined? Are these opaque to the user of this header?

Reply:
They're not opaque.  They're defined in this file.

Some of the structs in this file have members that are other structs.  This requires forward declarations of those structs.  To be consistent, I forward declared everything.